src/content/manager/ctp.manager.ts

```diff
@@ -4,6 +4,7 @@ import { ManagerConfig } from "../../interfaces/manager-config.interface";
 export abstract class CtpManager extends BaseManager {
     protected _content: any;
     protected _password: string;
+    protected _spaceKey?: string;
```

Comment:
Do we need to update the `DOCUMENTATION.md` for these changes?

Reply:
I am not so sure, as I don't see anything related to this CTP command in DOCUMENTATION.md.